### PR TITLE
Add tables for internal correlator file tracking

### PR DESCRIPTION
**Pull request by bhazelton**
## Description

Add tables to track the internal correlator file queues and the hand off to RTP.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Schema change (any change to the SQL tables)
- [ ] New feature without schema change (non-breaking change which adds functionality)
- [ ] Change associated with a change in redis structure
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Version change
- [ ] Build or continuous integration change
- [ ] Other

## Checklist:

Schema change:
- [x] My code follows the code style of this project.
- [x] I have added or updated the docstrings associated with my feature using the [numpy docstring format](https://numpydoc.readthedocs.io/en/latest/format.html).
- [x] I have created an alembic version file to produce the schema change.
- [x] I have tested looping upgrading and downgrading the alembic version and tests pass consistently.
- [x] I have updated or created monitoring scripts and daemons so any new tables will be automatically populated (if appropriate).
- [x] I understand the updates required onsite (detailed in the readme) and I will make those
changes when this is merged.
- [x] I have updated the schema documentation document with my changes (under docs/mc_definition.tex)
- [x] I have added tests to cover my new feature.
- [x] Unit tests pass **on site** (This is a critical check, CI can differ from site).
- [x] I have updated the [CHANGELOG](https://github.com/HERA-Team/hera_mc/blob/main/CHANGELOG.md).

### CODE REVIEW

File: hera_mc/correlator.py [hunk 1]
```diff
@@ -515,6 +529,243 @@ def _get_catcher_file_from_redis(redishost=DEFAULT_REDIS_ADDRESS, test_dict=None
...
+    time_start : BigInteger Column
+        GPS time (floored) that the end of day tag was first detected with a value of 0
+        (still converting).
+    time_converted : BigInteger Column
+        GPS time (floored) that the end of day tag was first detected with a value of 1
+        (finished converting, still uploading).
+    time_uploaded : BigInteger Column
+        GPS time (floored) that the end of day tag was first detected with a value of 2
+        (done uploading).
+    time_uploaded : BigInteger Column
```
Comment (bhazelton) [inline hera_mc/correlator.py:671]: wrong parameter name. Should be `time_launch_failed`

**Review by mkolopanis — COMMENTED** [review]
Seems good to me so far. let me know if you want to push on that coverage drop or just test on site.

**Comment by bhazelton** [comment]
most of the coverage issues were because the min_deps tests failed because I forgot to mark some tests as requiring redis. I just pushed a fix for the last remaining uncovered line, so I think the coverage is ok. I'll let you know when I've done some on-site testing.

**Comment by bhazelton** [comment]
@mkolopanis this has now been tested onsite. Also @dannyjacobs took a look and didn't have any suggestions, so I think it's ready for a re-review.

**Review by dannyjacobs — APPROVED** [review]
Looks good!

**Pull request merged**